saleor/graphql/order/mutations/order_grant_refund_create.py

```diff
@@ -289,11 +290,14 @@ def clean_input(
         requestor_is_app = info.context.app is not None
         requestor_is_user = info.context.user is not None and not requestor_is_app
 
+        site = get_site_promise(info.context).get()
```

Comment:
FYI: Forcibly evaluating a promise in a blocking manner is a Very Bad Idea (it forcibly re-enters the event loop) that leads to Saleor breaking when this happens during another promise evaluation (the event loop deadlocks and discards some of the unfulfilled promises).